### PR TITLE
Use Icons from Manifest on Layout

### PR DESCRIPTION
**Pull request by andreban**
- Refactored manifest.js to be a class.
- Added Icons and background colors to UI.
- Need to fetch the icons and serve from Gulliver server

### CODE REVIEW

**Comment by andreban** [comment]
@ithinkihaveacat PTAL

File: models/manifest.js [hunk 1]
```diff
@@ -16,26 +16,100 @@
...
+    }
+
+    return domain + path + iconUrl;
+  }
+
+  static fetch(manifestUrl, callback) {
+    const options = {
+      method: 'GET',
+      headers: {
+        'user-agent': [
```
Comment (juliantoledo) [inline models/manifest.js]: let's move the user agent to a const at the top.

**Comment by juliantoledo** [comment]
LGTM, Just one small comment.

**Comment by andreban** [comment]
Moved the user-agent to a static variable. @juliantoledo  PTAL.

**Comment by juliantoledo** [comment]
LGTM

**Pull request merged**